liblarch_gtk/__init__.py

```diff
@@ -17,6 +17,8 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------
 
+import gi
+gi.require_version("Gtk", "3.0")
 from gi.repository import Gtk, Gdk
 from gi.repository import GObject
```

Comment:
For passing pep8 checks, above two lines have to be commented with `# noqa`. BTW, these two lines could be joined into one `from ... import ...` for a little simplicity.